### PR TITLE
style(ui): Changed text colour for filter values

### LINKED ISSUE #423: Text colour for filter values too bright (hard to read)
(please test all portlets not just projects)

a picture is worth more than 100 words:

036 oss sw360 20170529 screenshot text coulour for filters

(let's make it black)

### PR DESCRIPTION
**Pull request by ChristophNiehoff**
closes #423

### CODE REVIEW

File: frontend/sw360-portlet/src/main/webapp/html/search/view.jsp [hunk 1]
```diff
@@ -44,7 +44,7 @@
             <tbody style="background-color: #f8f7f7; border: none;">
             <tr>
                 <td>
-                    <input type="text" maxlength="100" style="width: 200px; padding: 5px; color: gray;height:20px;"
```
Comment (alexbrdn) [inline frontend/sw360-portlet/src/main/webapp/html/search/view.jsp]: I'm not sure why `maxlength="100"` was there, but is there a reason why you removed it?

Comment (ChristophNiehoff) [inline frontend/sw360-portlet/src/main/webapp/html/search/view.jsp]: Yes, you are right. I copy/pasted too much. Changed it to include to maxlength again.

**Comment by alexbrdn** [comment]
@ChristophNiehoff , I took some liberty with editing the title and PR text above.

**Comment by mcjaeger** [comment]
tested it, required browser cache empty (of course), merged because it is very little change. works on the different tabs.

**Pull request merged**